rest/changestest/changes_api_test.go

```diff
@@ -576,48 +577,54 @@ func TestPostChangesAdminChannelGrantRemoval(t *testing.T) {
 	// Put several documents in channel PBS and HBO
 	mix1 := rt.PutDoc("mix-1", `{"channel":["PBS","HBO"]}`)
 	mix2 := rt.PutDoc("mix-2", `{"channel":["PBS","HBO"]}`)
-	mix3 := rt.PutDoc("mix-3", `{"channel":["PBS","HBO"]}`)
+	const mix3ID = "mix-3"
+	mix3Version := rt.PutDoc(mix3ID, `{"channel":["PBS","HBO"]}`)
 	mix4 := rt.PutDoc("mix-4", `{"channel":["PBS","HBO"]}`)
 
 	// Put several documents in channel ABC
 	_ = rt.PutDoc("abc-1", `{"channel":["ABC"]}`)
-	abc2 := rt.PutDoc("abc-2", `{"channel":["ABC"]}`)
-	abc3 := rt.PutDoc("abc-3", `{"channel":["ABC"]}`)
+	const abc2ID = "abc-2"
+	const abc3ID = "abc-3"
+	abc2Version := rt.PutDoc(abc2ID, `{"channel":["ABC"]}`)
+	abc3Version := rt.PutDoc(abc3ID, `{"channel":["ABC"]}`)
 	cacheWaiter.AddAndWait(13)
 
 	// Update some docs to remove channel
-	_ = rt.PutDoc("pbs-2", fmt.Sprintf(`{"_rev":%q}`, pbs2.Rev))
-	_ = rt.PutDoc("mix-1", fmt.Sprintf(`{"_rev":%q, "channel":["PBS"]}`, mix1.Rev))
-	_ = rt.PutDoc("mix-2", fmt.Sprintf(`{"_rev":%q, "channel":["HBO"]}`, mix2.Rev))
-	_ = rt.PutDoc("mix-4", fmt.Sprintf(`{"_rev":%q}`, mix4.Rev))
+	_ = rt.PutDoc("pbs-2", fmt.Sprintf(`{"_rev":%q}`, pbs2.RevID))
```

Comment:
This (and cases below) look like it should change to UpdateDoc and omit _rev in the body.